### ISSUE TITLE
Bug: Admin Summary Table issues

### ISSUE THREAD

**Issue by jmmcnj** [18F/dol-whd-14c#121]
Issues found in Admin table during review:

- [ ] Filter not firing
- [ ] spacing issues/cleanup UI
- [ ] Type values and statuses are not showing up in table
- [ ] dropdown arrows not showing up

**Comment by rrefoy** [comment]
Was this completed?

**Comment by jefferey** [comment]
Yes, this was wrapped up with PR #212. Sorry, forgot to move it over.

**Comment by mmurthydol** [comment]
@binwang89 wondering the reason behind this story moving from icebox to backlog?

**Comment by binwang89** [comment]
put it in backlog by a mistake, changed it back to icebox.
